server/monitor/src/test/java/org/apache/accumulo/monitor/it/WebViewsIT.java

```diff
@@ -123,18 +123,17 @@ public void testGetTablesConstraintViolations() {
    * a registered MessageBodyWriter capable of serializing/writing the object returned from your
    * endpoint. We're using a simple stubbed out inner class HashMapWriter for this.
    *
-   * @throws Exception not expected
    */
   @Test
-  public void testGetTablesConstraintPassing() throws Exception {
+  public void testGetTablesConstraintPassing() {
     // Using the mocks we can verify that the getModel method gets called via debugger
     // however it's difficult to continue to mock through the jersey MVC code for the properly built
     // response.
     // Our silly HashMapWriter registered in the configure method gets wired in and used here.
     Response output = target("tables/foo").request().get();
     assertEquals(200, output.getStatus(), "should return status 200");
     String responseBody = output.readEntity(String.class);
-    assertTrue(responseBody.contains("tableID=foo") && responseBody.contains("table=bar"));
+    assertTrue(responseBody.contains("tableId=foo") && responseBody.contains("table=bar"));
```

Comment:
could a constant be used for `tableId`?

Reply:
Yea I was thinking about this too. It seems like a good improvement. It would probably be helpful and make the code safer but would be a lot more changes. I'll merge this PR to get the test passing in the build and then create a follow on to add constants for all of the parameters in this category.